model/error.yaml

```diff
@@ -0,0 +1,34 @@
+groups:
+  - id: error
+    type: attribute_group
+    prefix: error
+    brief: >
+      This document defines the shared attributes used to
+      report an error.
+    attributes:
+      - id: description
+        brief: 'Describes a class of error operation has ended with.'
+        type:
+          allow_custom_values: true
+          members:
+            - id: empty
+              value: ""
+              brief: 'No error. Default value.'
```

Comment:
Do we specify what "default value" means anywhere? I remember using the phrase "if missing, can be assumed to be X" instead, if that is what is meant.

Reply:
Removed 'default' as it was causing confusion before, added a note that empty `error.id` and lack of it are the same